Factory/HttpFoundationFactory.php

```diff
@@ -80,10 +80,14 @@ private function getFiles(array $uploadedFiles)
      */
     private function createUploadedFile(UploadedFileInterface $psrUploadedFile)
     {
-        $temporaryPath = $this->getTemporaryPath();
-        $psrUploadedFile->moveTo($temporaryPath);
+        $temporaryPath = '';
+        $clientFileName = null;
```

Comment:
you could set this to `''`. below there is a check against null which only need to trigger if `$psrUploadedFile->getClientFilename()` returns null.